addons/l10n_ar/models/account_move.py

```diff
@@ -99,10 +109,8 @@ def _get_l10n_latam_documents_domain(self):
         if self.journal_id.company_id.account_fiscal_country_id.code == "AR":
             letters = self.journal_id._get_journal_letter(counterpart_partner=self.partner_id.commercial_partner_id)
             domain += ['|', ('l10n_ar_letter', '=', False), ('l10n_ar_letter', 'in', letters)]
-            codes = self.journal_id._get_journal_codes()
-            if codes:
-                domain.append(('code', 'in', codes))
-            if self.move_type == 'in_refund':
+            domain += self.journal_id._get_journal_codes_domain()
```

Comment:
Would be better to replace this part and user expressions, such as `expression.AND`

Reply:
@zaoral ping

Reply:
we change the approach to use expression.AND